drivers/volume/portworx/portworx.go

```diff
@@ -4649,3 +4649,23 @@ func init() {
 	torpedovolume.Register(DriverName, provisioners, &portworx{})
 	torpedovolume.Register(PureDriverName, csiProvisionerOnly, &pure{portworx: portworx{}})
 }
+
+//Update the pool label for a particular pool id
+func (d *portworx) UpdatePoolsLabels(n node.Node, poolID string, labels map[string]string) error {
+
+	labelsString := ""
+	for k, v := range labels {
+		labelsString += fmt.Sprintf("%s=%s,", k, v)
+	}
+	labelsString = strings.Trim(labelsString, ",")
+	cmd := fmt.Sprintf("%s sv pool update -u %s --labels=%s", d.getPxctlPath(n), poolID, labelsString)
```

Comment:
just a thought , instead of keeping -u its better to keep --uid ( just to maintain uniformity) 

nothing wrong in -u , you can ignore this comment if you wish

Reply:
I will take this up later when I am fixing the alerts portion for another bug